tests/test_config.py

```diff
@@ -643,92 +579,6 @@ def test_create_arg_parser(self, mock__print_message):
         output = self.parser.parse_args(["--verbose"])
         self.assertEqual(output.config_file, Path("./easy_sast.yml").absolute())
 
-        # Succeed when calling the create_arg_parser function and pass only
-        # --disable-auto-scan as an argument
-        output = self.parser.parse_args(["--disable-auto-scan"])
-        self.assertEqual(output.disable_auto_scan, True)
-
-        # Succeed and return the default value for disable-auto-scan when
-        # calling the create_arg_parser function without passing
-        # --disable-auto-scan as an argument
-        output = self.parser.parse_args(
-            ["--build-id=" + test_constants.VALID_UPLOAD_API["build_id"]]
-        )
-        self.assertEqual(output.disable_auto_scan, False)
-
-        # Succeed when calling the create_arg_parser function and pass only
-        # --disable-scan-nonfatal-modules as an argument
-        output = self.parser.parse_args(["--disable-scan-nonfatal-modules"])
-        self.assertEqual(output.disable_scan_nonfatal_modules, True)
-
-        # Succeed and return the default value for
-        # disable-scan-nonfatal-modules when calling the create_arg_parser
-        # function without passing --disable-scan-nonfatal-modules as an
-        # argument
-        output = self.parser.parse_args(["--disable-auto-scan"])
-        self.assertEqual(output.disable_scan_nonfatal_modules, False)
-
-        # Succeed when calling the create_arg_parser function and pass only
-        # --ignore-compliance-status as an argument
-        output = self.parser.parse_args(["--ignore-compliance-status"])
-        self.assertEqual(output.ignore_compliance_status, True)
-
-        # Succeed when calling the create_arg_parser function and pass only
-        # --ignore-compliance-status as an argument
-        output = self.parser.parse_args(["--disable-auto-scan"])
-        self.assertEqual(output.ignore_compliance_status, False)
-
-        # Succeed when calling the create_arg_parser function and pass only
-        # --version as an argument
-        with self.assertRaises(SystemExit) as contextmanager:
-            output = self.parser.parse_args(["--version"])
-        self.assertEqual(contextmanager.exception.code, 0)
-        # Using sys instead of _sys because it is the same thing per
-        # https://github.com/python/cpython/blob/6a263cf1adfc18cdba65c788dd76d35997a89acf/Lib/argparse.py#L90
-        mock__print_message.assert_called_once_with(veracode_version + "\n", sys.stdout)
```

Comment:
Since we're removing this from here entirely we should remove the `@patch` and `mock__print_message` from above.